### ISSUE TITLE
vita-toolchain fails to build on osx

### ISSUE THREAD

**Issue by xyzz** [vitasdk/buildscripts#11]
```
+ mkdir build-vita-toolchain
+ cd build-vita-toolchain
+ cmake /Users/user/build/src/vita-toolchain -DJansson_INCLUDE_DIR=/Users/user/build/build-native/vita-toolchain/install/include/ -DJansson_LIBRARY=/Users/user/build/build-native/vita-toolchain/install/lib/libjansson.a -Dlibelf_INCLUDE_DIR=/Users/user/build/build-native/vita-toolchain/install/include/ -Dlibelf_LIBRARY=/Users/user/build/build-native/vita-toolchain/install/lib/libelf.a -Dzlib_INCLUDE_DIR=/Users/user/build/build-native/vita-toolchain/install/include/ -Dzlib_LIBRARY=/Users/user/build/build-native/vita-toolchain/install/lib/libz.a -Dlibzip_INCLUDE_DIR=/Users/user/build/build-native/vita-toolchain/install/include/ -Dlibzip_LIBRARY=/Users/user/build/build-native/vita-toolchain/install/lib/libzip.a -DUSE_BUNDLED_ENDIAN_H=ON -DCMAKE_INSTALL_PREFIX=/Users/user/build/install-native -DDEFAULT_JSON=../share/db.json
-- The C compiler identification is Clang 3.1.0
-- The CXX compiler identification is Clang 3.1.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Could NOT find PkgConfig (missing:  PKG_CONFIG_EXECUTABLE)
-- Found Jansson: /Users/user/build/build-native/vita-toolchain/install/lib/libjansson.a
-- Could NOT find PkgConfig (missing:  PKG_CONFIG_EXECUTABLE)
-- Found libelf: /Users/user/build/build-native/vita-toolchain/install/lib/libelf.a
-- Could NOT find PkgConfig (missing:  PKG_CONFIG_EXECUTABLE)
-- Found zlib: /Users/user/build/build-native/vita-toolchain/install/lib/libz.a
-- Could NOT find PkgConfig (missing:  PKG_CONFIG_EXECUTABLE)
-- Found libzip: /Users/user/build/build-native/vita-toolchain/install/lib/libzip.a
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
libzip_CONFIG_INCLUDE_DIR
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src
   used as include directory in directory /Users/user/build/src/vita-toolchain/src

-- Configuring incomplete, errors occurred!
```

**Comment by ooPo** [comment]
It is looking for zipconf.h, try adding this to build-toolchain.sh, after line 269:

-Dlibzip_CONFIG_INCLUDE_DIR=$BUILDDIR_NATIVE/vita-toolchain/install/include/

**Comment by tplover** [comment]
Actually, it should be the following line since this is the location where zipconf.h is stored during compilation:

-Dlibzip_CONFIG_INCLUDE_DIR=$BUILDDIR_NATIVE/vita-toolchain/lib/libzip/include \

**Comment by xyzz** [comment]
fixed by e63dd4a765d30ca524b2bbeae3b0ccf7f983296c
